client/src/components/atoms/style/ScrollWindow.scss

```diff
@@ -1,6 +1,6 @@
 .scroll-window {
   width: 100%;
-  height: 100%;
+  max-height: 100%;
```

Comment:
The div containing ScrollWindow should set its max-height. Keep at height: 100% please (not max-height: 100%)